src/topp/SageAdapter.cpp

```diff
@@ -320,7 +320,21 @@ class TOPPSageAdapter :
     {
       enzyme_details = 
    R"("cleave_at": "")";
-    }           
+    }
+    else if (enzyme == "glutamyl endopeptidase")
```

Comment:
I don't like these magic strings. What if they change in EnyzmeDB?
This code should really be generated from the DB itself...